src/backend/columnar/cstore_tableam.c

```diff
@@ -1156,6 +1156,26 @@ cstore_tableam_finish()
 }
 
 
+/*
+ * Get the number of chunks filtered out during the given scan.
+ */
+int64
+ColumnarGetChunksFiltered(TableScanDesc scanDesc)
+{
+	CStoreScanDesc cstoreScanDesc = (CStoreScanDesc) scanDesc;
+	TableReadState *readState = cstoreScanDesc->cs_readState;
+
+	if (readState != NULL)
+	{
+		return readState->chunksFiltered;
+	}
+	else
+	{
+		return 0;
```

Comment:
when can `readState` be `NULL`? Is it possible at all? CI doesn't seem to be hit.

should we return a dedicated value here, sth like `-1` and interpret that as `unknown` for filtered chunks in the explain plan? If this can be reached I think having `0` in the explain plan is deceiving.